### PR TITLE
Consultation Feature: Edit (RemoveFromConsult)

### PR DESCRIPTION
**Pull request by wesho1107**
Create feature to remove students from a consult with specified index.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/RemoveFromConsultCommand.java [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+
+/**
+ * Removes a person identified using his/her name from the specified consultation by index.
+ */
+public class RemoveFromConsultCommand extends Command {
+    public static final String COMMAND_WORD = "removefromconsult";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Removes a person identified using his/her name from the specified consultation "
+            + "by index in the displayed consultation list.\n"
```
Comment (Originalidk) [inline src/main/java/seedu/address/logic/commands/RemoveFromConsultCommand.java]: Not sure if its just me or if the sentence is a bit odd/unclear. Think "Removes a person identified using his/her name from a consultation. The consultation is specified by index..." works is clearer and sounds better.

File: src/main/java/seedu/address/logic/commands/RemoveFromConsultCommand.java [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+            + "Parameters: INDEX (must be a positive integer) "
+            + PREFIX_NAME + "NAME\n"
+            + "Example: " + COMMAND_WORD + " 1" + PREFIX_NAME + "John Doe";
+
+    public static final String MESSAGE_SUCCESS = "Student(s) removed from consultation at index %1$s: %1$s";
+    public static final String MESSAGE_PERSON_NOT_FOUND = "No student(s) in address book matching given name(s)";
+    public static final String MESSAGE_NOT_FOUND_IN_CONSULT = "No student(s) in consultation matching given name(s)";
```
Comment (Originalidk) [inline src/main/java/seedu/address/logic/commands/RemoveFromConsultCommand.java]: Should be "matches" not matching

**Review by Originalidk — APPROVED** [review]
Other than a few nits, LGTM

File: src/main/java/seedu/address/logic/commands/RemoveFromConsultCommand.java (hunk 1 above)
Comment (wesho1107) [reply src/main/java/seedu/address/logic/commands/RemoveFromConsultCommand.java]: Good suggestion!

File: src/main/java/seedu/address/logic/commands/RemoveFromConsultCommand.java (hunk 2 above)
Comment (wesho1107) [reply src/main/java/seedu/address/logic/commands/RemoveFromConsultCommand.java]: Alright!

**Pull request merged**